src/jukebox/components/playermpd/__init__.py

```diff
@@ -231,6 +236,14 @@ def decode_2nd_swipe_option(self):
                                                          custom_action['args'],
                                                          custom_action['kwargs'])
 
+    def decode_end_of_playlist_next_action(self):
+        end_of_playlist_next_action = cfg.setndefault('playermpd', 'end_of_playlist_next_action', value='stop').lower()
+        if end_of_playlist_next_action in self.end_of_playlist_next_action_dict:
+            self.end_of_playlist_next_action = self.end_of_playlist_next_action_dict[end_of_playlist_next_action]
+        else:
+            logger.error(f'Config mpd.end_of_playlist_next_action must be one of '
+                         f'{self.end_of_playlist_next_action.keys()}. Ignore setting.')
```

Comment:
```suggestion
            self.end_of_playlist_next_action = self.end_of_playlist_next_action_dict['stop']
            logger.error(f"Config mpd.end_of_playlist_next_action must be one of "
                         f"{self.end_of_playlist_next_action.keys()}. "
                         f"Using default 'stop'")
```

Reply:
Maybe we can be a little bit more specific here. Thoughts?

Reply:
I've committed your suggestion.